grafana-plugin/src/containers/IntegrationSettings/parts/Autoresolve.tsx

```diff
@@ -56,7 +55,7 @@ const Autoresolve = ({ alertReceiveChannelId, onSwitchToTemplate, alertGroupId }
       'resolve_condition_template'
     );
     // @ts-ignore
-    if (autoresolveCondition == ['invalid template']) {
+    if (autoresolveCondition === ['invalid template']) {
```

Comment:
@maskin25 I believe this should get updated to
```tsx
if (autoresolveCondition === 'invalid template') {
```
according to our TS type for  the `resolve_condition_template` attribute [here](https://github.com/grafana/oncall/blob/1b3b0f9e13f2644d3edce0956112097f691c8f9a/grafana-plugin/src/models/alert_templates.ts#L16), and our database migration for this table [here](https://github.com/grafana/oncall/blob/1b3b0f9e13f2644d3edce0956112097f691c8f9a/engine/apps/alerts/migrations/0001_squashed_initial.py#L159), this attribute should indeed be a `string`, and never `string[]`